lib/util/urltils.js

```diff
@@ -162,6 +162,23 @@ module.exports = {
     }
   },
 
+  /**
+   * Obfuscates path parameters with regex from config
+   *
+   * @param {Config} config The configuration containing the regex
+   * @param {string} requestPath The Path to be obfuscated
+   * @returns {string} The obfuscated path
+   */
+  obfuscatePath: function obfuscatePath(config, requestPath) {
```

Comment:
i'd like to see a full suite of unit tests for this.  i know we didn't have any for the other urlutils before but should only need a few cases